va/va.c

```diff
@@ -229,36 +253,50 @@ void va_infoMessage(const char *msg, ...)
         n = vsnprintf(dynbuf, len + 1, msg, args);
         va_end(args);
         if (n == len)
-            va_log_info(dynbuf);
+            dctx->info_callback(dctx->info_callback_user_context, dynbuf);
         free(dynbuf);
     }
     else if (len > 0)
-        va_log_info(buf);
+        dctx->info_callback(dctx->info_callback_user_context, buf);
 #endif
 }
 
-static bool va_checkVtable(void *ptr, char *function)
+VADisplayContextP va_newDisplay(void)
```

Comment:
va_newDisplayContext for the name as it won't get confused with it returning a vaDisplayID